app/views/admin/suppliers/index.html.erb

```diff
@@ -1,8 +1,5 @@
-<% content_for :before_content, govuk_breadcrumbs(breadcrumbs: {
-  "Dashboard" => admin_dashboard_path,
-  "Manage suppliers" => nil
-}) %>
-<h1 class="govuk-heading-l">Manage suppliers</h1>
+<%= render "layout", locals={page: "suppliers"} %>
```

Comment:
I'm surprised rubocop didn't complain about {page: "suppliers"} not being spaced out correctly

Reply:
Doesn't lint erb files :/